README.md

```diff
@@ -274,7 +274,7 @@ localhost-key.pem localhost.pem
 3. Run the proxy with the following options
 
 ```bash
-go run main.go --ssl-cert-path=/tmp/localhost.pem --ssl-key-path=/tmp/localhost-key.pem --enable-ssl=true --kubeconfig=<YOUR KUBERNETES CONFIGURATION FILE>
```

Comment:
Why removing these lines?

TLS/SSL certificate is required when dealing with certificates-based authentication.

Reply:
Sorry, my fault. It was an oversight.
I'm going to close the PR.